### DISCUSSION TITLE
Having problem with getting the value of a select in RelationManager

### DISCUSSION THREAD

**Discussion by azim-kordpour** [filamentphp/filament#12030] [Help] [answered]
### Package

Panel builder

### Package Version

v3

### How can we help you?

Hi,
I'm new to Filamnet
In a relation manager, I want to show or hide an input based on the value of a select.

```php
 Select::make('type')
      ->live()
      ->options(SlideType::getLabels())
      ->default(SlideType::Exclusive->value),

TextInput::make('subtitle')
      ->visible(fn(Get $get) => dd($get('type')))
```

I just got null. I did the same process for creating a resource on other pages, and worked.
But in a relation manager, it does not work. do yo have an idea?

**Comment by leandrocfe** [comment]
Could you provide more info please? If you are using it in the `CreateAction`, it was supposed to work

```php
Select::make('type')
    ->live()
    ->options([
        'option1' => 'Option 1',
        'option2' => 'Option 2',
    ])
    ->default('option1'),

TextInput::make('subtitle')
    ->visible(fn (Get $get) => $get('type') === 'option1')
```

**Comment by azim-kordpour** [comment, marked as answer]
@leandrocfe 

It seems that `$get('type')` returns `null` for the first time when the form is loaded. 
Then, it returns the correct value. So, I added checking null to the condition to get the correct result.

**Reply by leandrocfe** [reply]
Use `filled($get('type'))`
